### PR TITLE
Add ReadMe

### PR DESCRIPTION
**Pull request by alan20854**
Sections with new content (compared to nightfall_dlp_action's readme):

**Example**

**Usage**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+**4. Allow Uncertified Orbs**
+To use our orb, make sure to allow uncertified orbs in Organization Settings. Instructions are [here](https://circleci.com/docs/2.0/orbs-faq/#using-uncertified-orbs). At this time, CircleCI still requires allowing un-certified orbs to use orbs developed by CircleCI Partners.
+## Supported GitHub Events
+The Nightfall DLP CircleCI Orb can run in a Circle Workflow triggered by the following events:
+
+- `PULL_REQUEST`
+- `PUSH`
+
+The Nightfall DLP CircleCI Orb is currently unable to be used in forked GitHub repositories due to GitHub's disabling of secrets sharing when Workflows originate from forks.
```
Comment (bradleymcallister97) [inline README.md:79]: Is this true for Circle CI since we are not using Github Secrets?

Comment (alan20854) [reply README.md:79]: Yea good question, I'll check to see if it works on a forked repo

**Comment by alan20854** [comment]
add section on optional user `BaseBranch` argument

File: README.md (hunk 1 above)
Comment (alan20854) [reply README.md:79]: So I forked nightfallai/nightfall_circle_orb as alan20854/nightfall_circle_orb.

With forked repos, if you (as the forker) add your personal access token (`GITHUB_TOKEN`) and a nightfall_api_key to your circle project (e.g. alan20854/nightfall_circle_orb), we can still get the diff and scan for findings. However, I don't believe it's possible to post these comments to the PR unless you have permissions to the original repository, which not every forker may have.

I can update this messaging to reflect that, and also add a ticket to the board for further upgrades to this orb.

**Review by bradleymcallister97 — APPROVED** [review]

**Review by bradleymcallister97 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+## Example
+Here's an example of the Nightfall DLP CircleCI Orb providing feedback on a Pull Request: 
+
+The orb runs when configured as a job in your CircleCI config:
+
+```yaml
+version: 2.1
+
+orbs:
+  nightfall_code_scanner: alan20854/testytest@dev:circle_service_bp #UPDATE THIS
```
[the remark this answers is not in the document]
Comment (henkejosh) [reply README.md:20]: update ^

File: README.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+## [Nightfall](https://nightfall.ai) DLP Orb: A code review tool that protects you from committing sensitive information to your repositories.
+
+The Nightfall DLP CircleCI Orb scans your code commits for sensitive information - like credentials & secrets, PII, credit card numbers & more - and posts review comments to your code hosting service automatically. The Nightfall DLP CircleCI Orb is intended to be used as a part of your CI to simplify the development process, improve your security, and ensure you never accidentally leak secrets or other sensitive information via an accidental commit.
+
+## Example
+Here's an example of the Nightfall DLP CircleCI Orb providing feedback on a Pull Request:
+![nightfall-dlp-circle-orb-example](https://cdn.nightfall.ai/nightfall-code-scanner/circleci-github-comments.png)
+
+Here's the CircleCI log output from the same run:
+![nightfall-dlp-circle-ui-example](https://cdn.nightfall.ai/nightfall-code-scanner/circleci-orb-workflow-ui.png)
```
Comment (henkejosh) [inline README.md:12]: can we get this on a newline?

**Pull request merged**